Test1.gb.gocd.yaml

```diff
@@ -131,4 +145,48 @@ pipelines:
             - docker
             tasks:
             - script: |
-                echo command to run smoke test 
\ No newline at end of file
+                echo command to run smoke test 
+  Capacity_testing:
+    group: AT20Group
+    label_template: ${COUNT}
+    lock_behavior: none
+    display_order: -1
+    environment_variables:
+      GIT_COMMIT_HASH: 1.2
+      TARGET_HOST: 192.168.0.46
+    materials:
+      Test1:
+        pipeline: Test1
+        stage: PackageAndPublish
```

Comment:
It should depend on the Previous pipeline (Acceptance.) if acceptance passes, then continue with the Capacity testing

Reply:
I completed the modification
![2023-06-13_11h11_28](https://github.com/AT20-DevOps/AT20_MEETING_SERVICE/assets/38434799/6142952c-eb38-406d-8e12-d1aa3595a811)